app/repositories/posts_repository.rb

```diff
@@ -0,0 +1,23 @@
+class PostsRepository
```

Comment:
And I guess best way - Create ApplicationBaseRepository or ApplicationRepository with base methods like 
create update find etc